shared/templates/sysctl/oval.template

```diff
@@ -122,81 +122,54 @@
-          <criterion comment="kernel static parameter {{{ SYSCTLVAR }}} set to {{{ COMMENT_VALUE }}} in /usr/lib/sysctl.d/*.conf"
-                    test_ref="test_{{{ rule_id }}}_static_usr_lib_sysctld"/>
-{{% endif %}}
-          <criterion comment="kernel static parameter {{{ SYSCTLVAR }}} set to {{{ COMMENT_VALUE }}} in /usr/local/lib/sysctl.d/*.conf"
-                    test_ref="test_{{{ rule_id }}}_static_usr_local_lib_sysctld"/>
-        </criteria>
-        <criterion comment="Check that {{{ SYSCTLID }}} is defined in at least one file" test_ref="test_{{{ rule_id }}}_not_defined"
-                   negate="true"/>
+        <criterion comment="kernel static parameter {{{ SYSCTLVAR }}} missing in sysctl files not managed by packages"
+                    test_ref="test_{{{ rule_id }}}_static_user_missing" />
+        {{% if MISSING_PARAMETER_PASS == "true" %}}
+          <criterion comment="{{{ "kernel static parameter"  + SYSCTLVAR + " set to " +
+                      COMMENT_VALUE + " or missing in sysctl files managed by packages" }}}"
+                    test_ref="test_{{{ rule_id }}}_static_pkg_not_wrong" />
+        {{% else %}}
+          <criterion comment="{{{ "kernel static parameter " + SYSCTLVAR + " set to " +
+                      COMMENT_VALUE + " in sysctl files managed by packages" }}}"
+                    test_ref="test_{{{ rule_id }}}_static_pkg_correct" />
+        {{% endif %}}
       </criteria>
-{{% if MISSING_PARAMETER_PASS == "true" %}}
-      <criterion comment="Check that {{{ SYSCTLID }}} is not defined in any file" test_ref="test_{{{ rule_id }}}_not_defined" />
     </criteria>
-{{% endif %}}
   </definition>
 
-  <ind:textfilecontent54_test id="test_{{{ rule_id }}}_not_defined" version="2"
+  <ind:textfilecontent54_test id="test_{{{ rule_id }}}_static_user_missing" version="1"
                               check="all" check_existence="none_exist"
                               comment="{{{ SYSCTLVAR }}} static configuration">
-    <ind:object object_ref="object_{{{ rule_id }}}_static_set_sysctls_unfiltered" />
+    <ind:object object_ref="object_static_user_{{{ rule_id }}}" />
   </ind:textfilecontent54_test>
 
-  <ind:textfilecontent54_test id="test_{{{ rule_id }}}_static" version="2"
-                              check="all" check_existence="any_exist"
+  <ind:textfilecontent54_test id="test_{{{ rule_id }}}_static_user" version="1"
+                              check="all" check_existence="all_exist"
                               comment="{{{ SYSCTLVAR }}} static configuration" state_operator="OR">
-    {{{ state_static_sysctld("sysctl") }}}
+    {{{ state_static_sysctld("user") }}}
   </ind:textfilecontent54_test>
 
-  <ind:textfilecontent54_test id="test_{{{ rule_id }}}_static_etc_sysctld" version="2" check="all"
+  <ind:textfilecontent54_test id="test_{{{ rule_id }}}_static_pkg_not_wrong" version="2" check="all"
```

Comment:
Could you wrap `test_{{{ rule_id }}}_static_pkg_not_wrong` and `test_{{{ rule_id }}}_static_pkg_correct` in Jinja conditionals checking for `MISSING_PARAMETER_PASS` just like in the `criterion`s?